### DISCUSSION TITLE
system/co/6

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#24] [Announcements]
# system/co/6

NoughtQ的笔记本

https://note.noughtq.top/system/co/6

**Comment by TillyEndless** [comment]
比较三种通信方式的开销  例子题目 答案好像放到习题里面了

**Reply by NoughtQ** [reply]
确实，当时截图是一页页截的，里面的内容我没有仔细看过，所以有了这个typo，感谢指出❤️，稍后我会修改的
